modules/core/src/lib/deck.ts

```diff
@@ -286,8 +286,8 @@ export default class Deck {
   readonly height: number = 0;
   // Allows attaching arbitrary data to the instance
   readonly userData: Record<string, any> = {};
-
-  protected canvas: HTMLCanvasElement | null = null;
+  readonly canvas: HTMLCanvasElement | null = null;
```

Comment:
This isn't going to work, as per the errors reported. @Pessimistress, what do you think about adding a `get canvas()` getter method?

Reply:
Good point - I reverted and added a helper similar to mapbox-gl's / maplibre's - also [exposed via react-map-gl](https://github.com/visgl/react-map-gl/blob/2bf7ca08a1782093f070b8f909d459efa7760777/src/types/lib.ts#L81).